qiskit/circuit/library/arithmetic/polynomial_pauli_rotations.py

```diff
@@ -140,7 +140,8 @@ class PolynomialPauliRotations(FunctionalPauliRotations):
 
     .. math::
 
-        |i\rangle |0\rangle \mapsto \cos(p(i)) |i\rangle |0\rangle + \sin(p(i)) |i\rangle |1\rangle
+        |i\rangle |0\rangle \mapsto \cos(\frac{p(i)}{2}) |i\rangle |0\rangle
+        + \sin(\frac{p(i)}{2}) |i\rangle |1\rangle
```

Comment:
It would be nice to scale the brackets to the size of the fraction 🙂 
```suggestion
        |i\rangle |0\rangle \mapsto \cos\left(\frac{p(i)}{2}\right) |i\rangle |0\rangle
        + \sin\left(\frac{p(i)}{2}\right) |i\rangle |1\rangle
```

Reply:
I've added like you suggested, thanks!! 👍